.github/workflows/python-ci.yml

```diff
@@ -10,6 +10,9 @@ on:
     types: [ opened, synchronize, reopened, ready_for_review ]
     branches: [ main, prod ]
 
+env:
+  COVIDCAST_INDICATORS_KEY: ${{ secrets.DELPHI_GITHUB_ACTIONS_EPIDATA_API_KEY }}
+
```

Comment:
do we need this?  do any of the tests run by github ci actually send requests to the production backend?

Reply:
nope. I'll remove this.